[DOCS] Exceptions flyout: update screenshots and procedure (#1619)
elastic/security-docs: docs/detections/detections-ui-exceptions.asciidoc

----
==============

. Do one of the following:
+
--
* To add an exception from the rule details page:
.. Go to the rule details page of the rule to which you want to add an
exception (*Detect* -> *Rules* -> *_<Rule name>_*).
.. Scroll down to the *Trend* histogram and select the *Exceptions* tab.
+
[role="screenshot"]
image::images/exception-histogram.png[Detail of Exceptions tab, 75%]
.. Click *Add new exception* -> *Add rule exception*.

* To add an exception from the Alerts table:
.. Go to *Detect* -> *Alerts*.

> .. Scroll down to the Alerts table, and from an alert for which you want to create an exception, click the *More Actions* button (*...*), then select *Add rule exception*.

Reviewer: ```suggestion
.. Scroll down to the Alerts table, go to the alert you want to create an exception for, click the *More Actions* button (*...*), then select *Add rule exception*.
```

Reviewer: Just a rec to simplify the middle part of this step.
